app/inbound/return/save-schedule.js

```diff
@@ -1,7 +1,11 @@
 const db = require('../../data')
+const isFirstPayment = require('./is-first-payment')
+
 const saveSchedule = async (settlement, transaction) => {
   if (settlement) {
-    await db.schedule.create({ settlementId: settlement.settlementId }, { transaction })
+    if (isFirstPayment(settlement.invoiceNumber)) {

```

Comment:
We only want to schedule original payment requests, not PPA.

Reply:
Incidentally this has the side effect of delivering another ticket where we need to supress interim top up payments.